### PR TITLE
An attempt to fix a broken build

### PR DESCRIPTION
**Pull request by agorina**

### CODE REVIEW

File: src/test/resources/application-it-test.yml [hunk 1]
```diff
@@ -2,6 +2,10 @@
 # Properties for integration tests.  Tests are run with an
 # active profile of 'it-test'
 # --------------------------------------------------------
+info:
+  app:
+    version: Test
```
Comment (agorina) [inline src/test/resources/application-it-test.yml]: We are smart to pull the version for the XML, and I did not know about it.

File: src/test/resources/items/mc-183563.qti.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+<qti-assessment-item xsi:schemaLocation="http://www.imsglobal.org/xsd/imsqtiasi_v3p0 https://purl.imsglobal.org/spec/qti/v3p0/schema/xsd/imsqti_asiv3p0_v1p0.xsd http://www.w3.org/1998/Math/MathML https://purl.imsglobal.org/spec/mathml/v3p0/schema/xsd/mathml3.xsd http://www.w3.org/2001/10/synthesis https://purl.imsglobal.org/spec/ssml/v1p1/schema/xsd/ssmlv1p1-core.xsd http://www.w3.org/2001/XInclude https://purl.imsglobal.org/spec/w3/2001/schema/xsd/XInclude.xsd " xmlns="http://www.imsglobal.org/xsd/imsqtiasi_v3p0" xmlns:m3="http://www.w3.org/1998/Math/MathML" xmlns:ssml11="http://www.w3.org/2001/10/synthesis" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" identifier="sbac-200-183563" title="SBAC Item 183563" tool-name="TIMS QTI 3.0 Export" tool-version="Test" time-dependent="false">
```
Comment (agorina) [inline src/test/resources/items/mc-183563.qti.xml]: `tool-version` is the change.

Comment (esotrope) [reply src/test/resources/items/mc-183563.qti.xml]: can each attribute have its own line. would be nice to setup the IDE formatter that way

Comment (agorina) [reply src/test/resources/items/mc-183563.qti.xml]: Interesting point, it would definitely help to clearly see the changes.
I will play with it.

**Pull request merged**